File: C#/Tiktok Login Design/Form1.cs
using System;

namespace Tiktok_Login_Design
{
    public partial class Form1
    {
        public Form1()
        {
            InitializeComponent();
        }
        private void Form1_Load(object sender, EventArgs e)
        {
            Guna2Panel1.Controls.Clear();
            Guna2Panel1.Controls.Add(new UserControlLogin());
        }

        private void Guna2ControlBox1_Click(object sender, EventArgs e)
        {
            Environment.Exit(0);
        }
    }
}

File: VB/Tiktok Login Design/Form1.vb
Public Class Form1
    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Me.Guna2Panel1.Controls.Clear()
        Me.Guna2Panel1.Controls.Add(New UserControlLogin())
    End Sub

    Private Sub Guna2ControlBox1_Click(sender As Object, e As EventArgs) Handles Guna2ControlBox1.Click
        End
    End Sub
End Class
